BraveRewardsUI/Tipping/TippingSelectionViewController.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import BraveRewards
+
+class TippingSelectionViewController: OptionsSelectionViewController {
+  
+  private let state: RewardsState?
+  
+  // Hide the super constructor with `DisplayableOption`
+  // in favour of our constructor with `BatValue`
+  private override init(options: [DisplayableOption],
+                        selectedOptionIndex: Int = 0,
+                        optionSelected: @escaping (_ selectedIndex: Int) -> Void) {
+    self.state = nil
+    super.init(options: options, selectedOptionIndex: selectedOptionIndex, optionSelected: optionSelected)
+  }
+  
+  init(state: RewardsState, options: [BATValue],
```

Comment:
I don't _love_ sending in `RewardsState` in here, and I know its needed for rates, but its fine for now. Though, do we need it to be a `RewardsState` or can we just send in a `BraveLedger` instead

Reply:
Fixed.